Source/PLCrashAsyncDwarfCFAStateEvaluation.cpp

```diff
@@ -227,11 +227,11 @@ plcrash_error_t dwarf_cfa_state<machine_ptr, machine_ptr_s>::eval_program (plcra
                 break;
                 
             case DW_CFA_def_cfa:
-                set_cfa_register(dw_expr_read_uleb128_regnum(), dw_expr_read_uleb128());
+                set_cfa_register(dw_expr_read_uleb128_regnum(), (machine_ptr_s)dw_expr_read_uleb128());
```

Comment:
Here (and elsewhere below), shouldn't `machine_ptr` should be used when operating on unsigned DWARF CFA register values?

Reply:
Fixed